qplist.pro

```diff
@@ -0,0 +1,14 @@
+QT += xml
+TEMPLATE = lib
+
+CONFIG += staticlib flat
```

Comment:
Why do I want CONFIG+=flat?  The qmake docs say this only effects the vcapp template, and that it's turned on by default anyway.